packages/less/src/less-browser/index.js

```diff
@@ -46,42 +50,32 @@ export default (window, options) => {
         return cloned;
     }
 
-    // only really needed for phantom
-    function bind(func, thisArg) {
-        const curryArgs = Array.prototype.slice.call(arguments, 2);
-        return function() {
-            const args = curryArgs.concat(Array.prototype.slice.call(arguments, 0));
-            return func.apply(thisArg, args);
-        };
-    }
-
     function loadStyles(modifyVars) {
         const styles = document.getElementsByTagName('style');
-        let style;
 
-        for (let i = 0; i < styles.length; i++) {
-            style = styles[i];
+        for (let style of styles) {
             if (style.type.match(typePattern)) {
-                const instanceOptions = clone(options);
-                instanceOptions.modifyVars = modifyVars;
+                const instanceOptions = {
+                    ...options,
```

Comment:
This will perform a shallow copy of options, instead of a deep clone... I'm not sure if that will be an issue or not 🤔  ... but I would avoid it without documented types.